client/payment-methods/index.js

```diff
@@ -50,6 +52,10 @@ const availableMethods = [
 ];
 
 const PaymentMethods = ( { enabledMethodIds, onEnabledMethodIdsChange } ) => {
+	const [
+		showPaymentMethodsSelector,
```

Comment:
(minor) What do you think about using `is`/`has` prefixes on boolean? In this case maybe `isPaymentMethodsSelectorVisible` or `isPaymentMethodsModalVisible` or `isPaymentMethodsSelectorModalVisible`?
It's definitely more verbose, tho 🤷  but I'm thinking it might help with consistency

Reply:
Sure thing! Sounds good to me. I haven't got yet the hold of the React naming conventions, so I really appreciate your feedback on this matter. :bow: